MEMBER_TRAVEL_FUND.md

```diff
@@ -12,6 +12,60 @@ and the Node.js Foundation projects.
 
 Any member can apply for travel funds.
 
+If you need a visa for the country in which the Node.js event you are attending
+is being held, see the next section titled "Prepare for visa application (optional)".
+If you don't need a visa, skip that section.
+
+A quick way to check if you may need a visa is to enter `wiki visa requirements
+for [your country here] citizens` into a search engine. The first result is typically a
+wikipedia page with a table of all countries' visa policy that would apply to you.
+
+### Prepare for visa application (optional) 🛂
+
+If you need a visa, don't worry! You can still apply for travel funds. You'll need
+to account for extra time to prepare your visa application, apply for the visa,
+and receive it before the date of travel.
+
+We've provided here a checklist of items that a visa application could require.
+Thinking about how long it will take you to prepare these will give you an idea of how
+long it could take to prepare your visa application.
+
+Typical visa applications may ask for any or all of the following:
+
+* An original passport. If you don't have one, you'll need to apply for one first.
+Check with your passport issuing authority for the processing time.
+* Passport-sized photos.
+* An original proof of residency for the country where you live,
+assuming you are not a citizen. For example, if you live in Canada this could be
+a Permanent Residence Card or a Temporary Residence Visa.
+* Copies of any of the above documents.
+* A copy of your air ticket.
+* A copy of your accommodation reservations.
```

Comment:
I used to have my accommodation and flights booked by the foundation staff when I spoke at NINA for the first time, and they offered help when I spoke there for the second time. Maybe it's also possible to request help from the foundation staff when you are not a speaker but is funded by the foundation to travel?